SVGViewTests/SVGRefGenerator.swift

```diff
@@ -129,6 +130,7 @@ class SVGRefGenerator: XCTestCase {
         createReference(name: "paint-color-03-t", version: v12)
         createReference(name: "paint-color-201-t", version: v12)
         createReference(name: "paint-fill-04-t", version: v12)
+        createReference(name: "paint-fill-06-t", version: v12)
```

Comment:
What's in this?

Reply:
The [test](https://github.com/GoodNotes/SVGView/blob/633735218e30a361d067b451d5ef392a2197c152/SVGViewTests/SVG12Tests.swift#L61) is present in the orginally repo. We need this line to regenerate the ref file.